psydac/linalg/block.py

```diff
@@ -305,7 +306,8 @@ def conjugate( self, out=None):
             Lij.conjugate(out=Lij_out)
         out._sync = self._sync
         return out
-    def conj( self, out=None):
+
+    def conj(self, out=None):
```

Comment:
> Good question. I think that `conj` should be provided by the `Vector` base class.

You mean that the `conj` method should only be define in the `Vector` base class as a function calling the `conjugate` method ?

Reply:
If I understood correctly, it should be like this
[16541b0](https://github.com/pyccel/psydac/pull/273/commits/16541b078ca56b855b19d0a4f0d7374751d0755f)

Reply:
I guess that makes sense, but then we should have an abstract method `conjugate` in basic.Vector so that every Vector subclass must implement conjugate.

Reply:
It's already the case in our actual code for `Vector`.